apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -5,11 +5,18 @@ import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className?: string;
```

Comment:
```suggestion
```

Reply:
You should remove className from Properties. Try to make the component customizable without adding className prop